### ISSUE TITLE
export to csv empty

### ISSUE THREAD

**Issue by gingerlime** [WeAreWizards/passopolis-server#13]
Hi there passopolis,

I tried to export secrets to CSV. The export is generated and downloaded, but the file is empty, besides one line with something that looks like a template ...

```
url,username,password,extra,name,grouping,fav
http://sn,abc,xyz,note,test secret,Secure Notes,0
```

Not sure if the problem is with the extension or the server, but just wanted to report this and check with you.

Thanks for keeping passopolis up and running.

**Comment by teh** [comment]
Hi - can you see a list of secrets below the "Secrets to Export" headline? I can't reproduce (I just exported my secrets). Do you have anything else unusual, e.g. an old Chrome or secrets shared across accounts?

**Comment by gingerlime** [comment]
I was using it on Chrome 65/Linux, and the secrets were showing below, but the export was empty/strange as I described. 

We do share secrets. That's the main feature of passopolis that makes it so great :)

Tried again on Chrome 65 on Mac and on Linux, and the export worked :) 👍 so probably it was some temporary glitch of some sort. 

Sorry for the false alarm, and thanks for your help. I'll close this.

p.s. any chance for a Firefox addon? :)
